packages/frontend/src/components/send/components/InputAccountId.js

```diff
@@ -171,7 +174,15 @@ class InputAccountId extends Component {
 
         try {
             await checkAvailability(accountId);
-            setAccountIdIsValid(true);
+            if (HAPI_RISK_SCORING) {
```

Comment:
Seems like In case of HAPI service failure "does not exists while viewing" will be shown?

Reply:
outdated